src/components/schedule-page/corrupted-schedule.component.tsx

```diff
@@ -21,12 +21,24 @@ export function CorruptedScheduleComponent(): JSX.Element {
   const { past } = useSelector(
     (state: ApplicationStateModel) => state.actualState.persistentSchedule
   );
-  const isPreviousVersionAvailable = past.length > MINIMUM_UNDO_COUNT_TO_REVERT_NORMAL_SCHEDULE;
+  const { month_number: month, year } = useSelector(
+    (state: ApplicationStateModel) => state.actualState.persistentSchedule.present.schedule_info
+  );
+  const currentMonthPastRevisions = past.filter(
+    (schedule) =>
+      schedule.schedule_info.month_number === month && schedule.schedule_info.year === year
+  );
+  const isPreviousVersionAvailable =
+    currentMonthPastRevisions.length >= MINIMUM_UNDO_COUNT_TO_REVERT_NORMAL_SCHEDULE;
```

Comment:
Dlaczego teraz jest `>=`?
Czy nie jest tak, że my powinniśmy usunąć ten grafik który spowodował to, że pojawił się problem + uwzględniamy to, że miesiąc nie jest `autoGenerated`?
Moim zdaniem powinno być `> 2`

Reply:
Ok
Rozumiem, że to przez to, że teraz w `past` nie ma `isAutoGenerated`.
![image](https://user-images.githubusercontent.com/21079319/111001018-5b42f000-8383-11eb-9863-f551299dad3f.png)



W takim razie potrzeba też usunąć ten `if`
![image](https://user-images.githubusercontent.com/21079319/111001261-bffe4a80-8383-11eb-8472-47c3d5d596ba.png)

Reply:
Ok
To nie było związane ze stanem, ale z tym, że kiedy grafik się psuł, to potrzeba było usunąć grafik który doprowadził do błędu oraz ten sam grafik, ale ze zmienioną flagą.

![image](https://user-images.githubusercontent.com/21079319/111001506-2f743a00-8384-11eb-977f-31bf5505afef.png)

Ty tego nie zmieniałeś, więc moim zdaniem dalej powinno być `>`